test/plugins/features/movies/controller.test.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+
+const Knex = require('../../../../lib/libraries/knex.js');
+
+const Controller = require('../../../../lib/plugins/features/movies/controller');
+const Movie = require('../../../../lib/models/movie.js');
+
+describe('movie controller', () => {
+
+  beforeEach('clear tables before test start', async () => {
+
+    await Knex.raw('TRUNCATE movies CASCADE');
+
+  });
+
+  describe('create', () => {
+
+    it('creates a movie', async () => {
+      const payload = { title: 'WALL-E' };
+      const movie = await Controller.create(payload);
+      expect(movie.get('title')).to.eql(payload.title);
+
+    });
+
+  });
+
+  describe('get all movies', () => {
```

Comment:
I'd say that you should group all of these get functions rather than have a describe block for each way that the GET function is used

Reply:
That makes sense! 👍